src/main/java/org/fundacionjala/coding/roger/FizzBuzz.java

```diff
@@ -0,0 +1,44 @@
+package org.fundacionjala.coding.roger;
+
+import java.util.StringJoiner;
+/**
+ * Kata 8 by roger.
+ *
+ */
+public class FizzBuzz {
+    private static final int FIVE = 5;
+    private static final int THREE = 3;
+    private static final int FIFTEEN = 15;
+    /**
+     *@param number input String.
+     *@return return the string.
+     */
+    public String fizzBuzzNumbers(final int number) {
+        StringJoiner sj = new StringJoiner("\n");
+        for (int i = 0; i < number; i++) {
+            sj.add(fizzBuzzNumber(i + 1));
+        }
+        System.out.println(sj.toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)